### DISCUSSION TITLE
Improvement: Proposed name change of FTF -> GBTS seeding

### DISCUSSION THREAD

**Discussion by Rosie-Hasan** [acts-project/acts#2943] [General]
Change naming of new seeding algorithm from FTF (FastTrackFinder) to GBTS (Graph Based Track Seeding) as this is a clearer desription of the algorithm. 
Code implemented in PRs #2227 #2726 and more details given presented at [ACTS-ITK presention ](https://indico.cern.ch/event/1321208/#20-update-on-ftf-implementatio).

The proposed name changes for file names and classes are listed below. I would apprecaite any opinions or feedback on these before the 20th of December, when I will submit a PR of these changes. 

- `SeedFinderFTF` -> `GBTSSeedFinder` (or `SeedFinderGBTS`?) 
- `GNN_DataStorage` -> `GBTSDataStorage` 
- `GNN_Geometry` -> `GBTSGeometry`
- `SeedingFTFAlgorithm` -> `GBTSSeedingAlgorithm` (or `SeedingGBTSAlgorithm`?) 
- `FasTrackConnector`-> `GBTSConnector` 
- `TrigBase` -> `GBTSBase` 
- `full_chain_itk_FTF.py` -> `full_chain_itk_GBTS.py`

**Comment by paulgessinger** [comment]
My only request is to write `Gbts` instead of `GBTS` in class names, i.e. `GbtsConnector`.

**Comment by CarloVarni** [comment]
Comments from my side:
- I Agree with @paulgessinger `GBTS` -> `Gbts`
- I'd follow the pattern (for this and any seeding algorithm) for the seeding alg name: `<TYPE>SeedingAlgorithm`. So I have a preference indeed for `GbtsSeedingAlgorithm`
- For the seed finder I see other implementations adopt the convention (opposite to the algorithm convention ... maybe in the future we may impose a common pattern for all the Acts Code but not immediately): `SeedFinder<TYPE>`
- Avoid `_` in classes names

**Comment by Rosie-Hasan** [comment]
Thank you both, I have made the name changes based on these suggestions and will submit a PR soon
